samples/ScratchIslandApp/SampleApp/init.cpp

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation
+// Licensed under the MIT license.
+
+#include "pch.h"
+#include <LibraryResources.h>
+#include <WilErrorReporting.h>
+
+// Note: Generate GUID using TlgGuid.exe tool
+TRACELOGGING_DEFINE_PROVIDER(
+    g_hSampleAppProvider,
+    "Microsoft.Windows.Terminal.App",
+    // {24a1622f-7da7-5c77-3303-d850bd1ab2ed}
+    (0x24a1622f, 0x7da7, 0x5c77, 0x33, 0x03, 0xd8, 0x50, 0xbd, 0x1a, 0xb2, 0xed),
+    TraceLoggingOptionMicrosoftTelemetry());

```

Comment:
you can probably kill this one

Reply:
or replace it with a new one generated with tlgguid

Reply:
Yes, please do not re-use it.